src/openzaak/components/documenten/admin.py

```diff
@@ -251,3 +273,22 @@ def get_object_actions(self, obj):
             link_to_related_objects(Gebruiksrechten, obj.canonical),
             link_to_related_objects(ObjectInformatieObject, obj.canonical),
         )
+
+    def get_urls(self):
+        urls = super().get_urls()
+        my_urls = [
+            path(
+                "<int:pk>/download/",
+                self.download,
+                name="documenten_enkelvoudiginformatieobject_download",
+            ),
+        ]
+        return my_urls + urls
+
+    def download(self, request, pk):
+        obj = self.get_object(request, pk)
+        response = FileResponse(obj.inhoud)
```

Comment:
You _really_ don't want to have Django serving potentially massive files from disk. This should use `sendfile` on nginx, all of this is implemented in django-privates already. Please look up the documentation/implementation of django-privates on how to customize the attachment filename.